### DISCUSSION TITLE
[Solved] i have problem with my Relog.Yes it auto relog me into sever but i cant add action to auto send me to my /home farm and start bot

### DISCUSSION THREAD

**Discussion by Aquanutt** [ORelio/Minecraft-Console-Client#2445] [Q&A] [answered]
first i think just add script "recon.txt" in last sentence in relog func 
[ChatBot.AutoRelog]
Enabled = true
Delay = { min = 3.0, max = 3.0 }            # The delay time before joining the server. (in seconds)
Retries = 3                                 # Retries when failing to relog to the server. use -1 for unlimited retries.
Ignore_Kick_Message = true                  # When set to true, autorelog will reconnect regardless of kick messages.
# If the kickout message matches any of the strings, then autorelog will be triggered.
Kick_Messages = [ "connection has been lost", "server is restarting", "server is full", "too many people", "disconnected by server", ]
Action = "script reconnect.txt" 

and in my roconnect file have
send /Login pass
wait 15
send /skyblock
wait 15
send /home mining
wait 35
digbot start

How to make it not just auto reconnect when disconnect but also send me to my specific location

**Comment by Aquanutt** [comment, marked as answer]
but in [[ChatBot.ScriptScheduler.TaskList]]
Task_Name = "Task Name 1"
Trigger_On_First_Login = true
Trigger_On_Login = false
Trigger_On_Times = { Enable = false, Times = [ 14:00:30, ] }
Trigger_On_Interval = { Enable = false, MinTime = 3.6, MaxTime = 4.8 }
Action = "script login1.txt"
i do like this i its work but when i add Action in relog it ,the Action = "script reconnect.txt" just gone

**Reply by milutinke** [reply]
AutoRelog does not support executing actions on reconnect, use the Script Scheduler chat bot with trigger on login.

**Reply by Aquanutt** [reply]
Thx sir i finally can make it work
